charts/sourcegraph/values.yaml

```
@@ -132,7 +132,7 @@ cadvisor:
   serviceAccount:
     # -- Enable creation of ServiceAccount for `cadvisor`
     create: true
-    # -- Name of the ServiceAccount
+    # -- Name of the ServiceAccount to be created or an existed ServiceAccount
```

```suggestion
    # -- Name of the ServiceAccount to be created or an existing ServiceAccount
```
etc etc throughout the values file